eZ/Publish/Core/Base/Container/Compiler/Search/SignalSlotPass.php

```diff
@@ -20,31 +19,47 @@
  */
 class SignalSlotPass implements CompilerPassInterface
 {
+    /**
+     * Name of a search engine for which Compiler Pass was instantiated.
+     *
+     * @var string|null
+     */
+    private $searchEngineAlias;
+
+    public function __construct($searchEngineAlias = null)
```

Comment:
Hmm, then it's better that we introduce a new class and deprecate this one _(otherwise leave it as is)_, because the logic further down expects it to be set _(and it should)_.